components/homme/test/reg_test/run_tests/test-list.cmake

```diff
@@ -9,7 +9,6 @@ SET(HOMME_TESTS
   baro2c.cmake
   baro2d.cmake
   baroCamMoist.cmake
-  baroCamMoist-SL.cmake
```

Comment:
Is this PR removing the baroCamMoist-SL test?

Reply:
Only without compose build. So, should run on our machines, we don't run routinely on gpus.

Reply:
Right, line 27 of this file appends baroCamMoist-SL.cmake to the list if HOMME_ENABLE_COMPOSE=TRUE.